The Open Sourcing and refactoring of BioKanga on GitHub has resulted in some build issues. This
patch removes references to code which nolonger exists in refactored code of v4.3.4
author: Nathan S. Watson-Haigh (University of Adelaide)
--- ./configure.ac	2017-04-04 11:47:18.067209892 +0930
+++ ./configure.ac	2017-04-04 11:47:46.719519662 +0930
@@ -23,11 +23,10 @@
 	 fasta2dist/Makefile locsw/Makefile quickcount/Makefile csv2feat/Makefile
 	 genalignref2relloci/Makefile genalignloci2core/Makefile maploci2features/Makefile
 	 processcsvfiles/Makefile genalignconf/Makefile Loci2Phylip/Makefile blast2csv/Makefile genrollups/Makefile
-	 gengoterms/Makefile gengoassoc/Makefile goassoc/Makefile
 	 genWiggle/Makefile gencentroidmetrics/Makefile proccentroids/Makefile DNAseqSitePotential/Makefile SimulateMNase/Makefile
 	 filterreads/Makefile prednucleosomes/Makefile RNAseqSitePotential/Makefile
 	 genDESeq/Makefile fasta2struct/Makefile kangarg/Makefile genultras/Makefile
- 	 LocateROI/Makefile genstructprofile/Makefile genstructstats/Makefile predconfnucs/Makefile gennucstats/Makefile fastafilter/Makefile
+ 	 LocateROI/Makefile genstructprofile/Makefile predconfnucs/Makefile gennucstats/Makefile fastafilter/Makefile
 	 BEDMerge/Makefile BEDFilter/Makefile genElementProfiles/Makefile genzygosity/Makefile genGenomeFromAGP/Makefile genmarkers/Makefile
 	 HammingDist/Makefile genreadsde/Makefile ufilter/Makefile uhamming/Makefile genseqcandidates/Makefile GFFfilter/Makefile loci2dist/Makefile
          genNormWiggle/Makefile GTFfilter/Makefile genDiffExpr/Makefile dmpbioseq/Makefile genpseudogenome/Makefile usimdiffexpr/Makefile RNAFragSim/Makefile mergeoverlaps/Makefile
--- Makefile.am	2017-04-04 11:51:40.878045716 +0930
+++ Makefile.am	2017-04-04 11:51:57.518224861 +0930
@@ -8,10 +8,9 @@
 	  genalignref2relloci genalignloci2core maploci2features \
 	  processcsvfiles genalignconf Loci2Phylip blast2csv genrollups \
 	  genWiggle proccentroids gencentroidmetrics SimulateMNase predconfnucs \
-	  gengoterms gengoassoc goassoc \
 	  DNAseqSitePotential RNAseqSitePotential prednucleosomes gennucstats \
           uhamming LocateROI genDESeq genGenomeFromAGP kangarg genultras \
-	  genstructprofile genstructstats filterreads fasta2struct ufilter genseqcandidates genmarkers GFFfilter \
+	  genstructprofile filterreads fasta2struct ufilter genseqcandidates genmarkers GFFfilter \
 	  genElementProfiles BEDMerge BEDFilter loci2dist genzygosity genNormWiggle GTFfilter genDiffExpr \
           dmpbioseq genpseudogenome usimdiffexpr genreadsde HammingDist RNAFragSim mergeoverlaps fastafilter \
           kangasr kangar kangahrdx kangade kangas kangapr csv2sqlite biokanga locmarkers PEscaffold SSRdiscovery pacbiokanga
